libbeat/generator/fields/fields.go

```diff
@@ -39,27 +32,54 @@ type YmlFile struct {
 	Indent int
 }
 
-func collectBeatFiles(beatPath string, fieldFiles []*YmlFile) ([]*YmlFile, error) {
-	commonFields := filepath.Join(beatPath, "_meta", "fields.common.yml")
-	_, err := os.Stat(commonFields)
-	if os.IsNotExist(err) {
-		return fieldFiles, nil
-	} else if err != nil {
-		return nil, err
+func collectCommonFiles(esBeatsPath, beatPath string, fieldFiles []*YmlFile) ([]*YmlFile, error) {
+	commonFields := []string{
+		filepath.Join(beatPath, "_meta", "fields.common.yml"),
 	}
 
-	files := []*YmlFile{
-		{
-			Path:   commonFields,
+	var libbeatFieldFiles []*YmlFile
+	var err error
+	if !isLibbeat(beatPath) {
+		commonFields = append(
+			[]string{
+				filepath.Join(esBeatsPath, "libbeat", "_meta", "fields.common.yml"),
+				filepath.Join(beatPath, "_meta", "fields.yml"),
```

Comment:
I think _meta/fields.yml is not used anymore.

Reply:
I found a place that has some old references to fields files that needs updated to point to plain old fields.yml.

https://github.com/elastic/beats/blob/5443c8ac37ca333342611605690df13a214b128f/libbeat/tests/system/beat/beat.py#L462-L496